### ISSUE TITLE
Support starting the replicator enabled

### ISSUE THREAD

**Issue by per-samuelsson** [Starcounter/Replicator#4]
Right now, when the sample replicator start, it's not enabled. I have to manually specify it's enabled in the UI for it to run.

I think its justified to support the simple feature of enabling the replicator when started right out of the box, not having every fork having to do so.

```
star bin\debug\replicator.exe --enabled
```

@linkdata, let me know if it's ok for me to implement this.

**Comment by linkdata** [comment]
Mmmm... command-line parameters. Just the thing when you *know* the UI was written by a sadistic monkey on hallucinogens (i.e., me). Sure, go ahead. If you feel ambitious, please add `--parent [ip:port]` as well.

**Comment by per-samuelsson** [comment]
> Sure, go ahead.

Cool. Will do.

> If you feel ambitious, please add `--parent [ip:port]` as well.

That's a bit different, since the parent is normally configured (in the database). When would the switch do - override the configured value for the starting session? Update the configuration? Fail if there was a conflict (i.e. the parent was already configured)?

I suggest we postpone that.

**Comment by per-samuelsson** [comment]
Done. Merged and pushed in dbd6588267b590d910d440d63b1dd5501101a317.

I forgot we reserve `--[key]` for `star.exe`, so I went for `@enabled`, i.e:

```
star bin\debug\replicator.exe @enabled
```
